client/src/components/Toasts/Toasts.scss

```diff
@@ -0,0 +1,101 @@
+.toasts {
+  position: fixed;
+  top: 20px;
+  right: 20px;
+  width: 250px;
+  z-index: 9999;
+
+  &__notice-wrap {
+    position: absolute;
+  }
+}
+
+
+@keyframes toastFadeIn {
```

Comment:
I'm using keyframe animation instead of the jQuery animate method in the plug-in. This should make it smoother.